AppointmentReminders.Web/Scripts/bootstrap-datetimepicker.js

```diff
@@ -0,0 +1,2426 @@
+/*! version : 4.14.30
```

Comment:
@acamino actually I just checked the document outlining the requirements and they require serving from CDN instead of local files. They specifically mention not to check in libraries into the repo.